src/modules/rover_pos_control/RoverPositionControl.hpp

```diff
@@ -233,12 +246,45 @@ class RoverPositionControl final : public ModuleBase<RoverPositionControl>, publ
 	void		manual_control_setpoint_poll();
 
 	/**
-	 * Control position.
+	 * @brief Generate rate setpoint based on desired position setpoint using L1 guidance
+	 *
+	 * If the speed control mode (`GND_SP_CTRL_MODE`) is set to 1, PID velocity controller is used for
+	 * closed-loop control. Otherwise, throttle is fixed at position setpoint trpilet's cruising throttle
+	 * setting (open-loop control).
+	 *
+	 * @param global_pos [deg] Lattitude & Longitude of the vehicle currently
+	 * @param ground_speed [m/s] Global velocity in North - East - Down frame
+	 * @param _pos_sp_triplet Modified version of position setpoint triplet generated by the Navigator, indicating desired position
```

Comment:
I added function descriptions with units strictly specified to avoid future confusions on the unit being used. Also, having some comment helps :wink: